### PR TITLE
Fix for #33634  - Prevent overrides being wiped during upgrade to PS8

### PR DESCRIPTION
**Pull request by gennaris**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Prevent existing overrides from being wiped during upgrade to PS8
| Type?             | bug fix
| BC breaks?        | n/a
| Deprecations?     | n/a
| Fixed ticket?     | Fixes #33634  ( https://github.com/PrestaShop/PrestaShop/issues/33634 )
| How to test?      | Upgrade from PS 1.7.x to PS 8.x having some overrides - compare override folder before / after this PR

### CODE REVIEW

**Review by kpodemski — DISMISSED** [review]

**Comment by kpodemski** [comment]
thank you @gennaris :)

File: classes/UpgradeTools/FileFilter.php [hunk 1]
```diff
@@ -145,6 +145,10 @@ public function getFilesToIgnoreOnUpgrade()
             '/app/config/parameters.yml',
             '/install',
             '/install-dev',
+            '/override',
+            '/override/classes',
+            '/override/controllers',
+            '/override/modules'
```
Comment (kpodemski) [inline classes/UpgradeTools/FileFilter.php:151]:
```suggestion
            '/override/modules',
```

**Review by aniszr — APPROVED** [review]
Hello @gennaris 

I tested your PR and the described improvements are verified !

- Autoupgrade from PS 1.7.8.9 to PS 8.1.1 **without PR**

-> In the **override** folder there was at the bigenning 5 folders (classes,controllers,modules,index.php and .htaccess)
-> After the upgrade was done the **override** folder contain only 2 files (index.php and .htaccess)

- Autoupgrade from PS 1.7.8.9 to PS 8.1.1 **with PR**

->The **override** folder kept all his 5 files (classes,controllers,modules,index.php and .htaccess) after the upgrade was done

See attached screenrecord:

https://github.com/PrestaShop/autoupgrade/assets/92626242/9963a94d-a7a4-4012-9c47-d0a841180a72

QA :heavy_check_mark: 

Thank you!

**Comment by kpodemski** [comment]
thanks @gennaris 🎉

**Comment by MatShir** [comment]
🥳

**Pull request merged**